examples/logger.py

```diff
@@ -85,11 +117,21 @@ def __init__(self, name: str, parent: Optional[QObject] = None):
         self.loggerFrame.clearButton.clicked.connect(self.checkToClear)
         self.confirmFrame = ConfirmClearingFrame()
         self.confirmFrame.confirmed.connect(self.clearLog)
+        # define handler
+        self.handler = LoggingHandler(self.addLog)
+        # arbitrary format
+        fs ="%(name)s %(message)s"
+        formatter = logging.Formatter(fs)
+        self.handler.setFormatter(formatter)
+        logger = logging.getLogger()
+        logger.addHandler(self.handler)
+        logger.setLevel(logging.DEBUG)
```

Comment:
You don't have to set the log level here. Let's set this later when you implement the level selector GUI.
It is okay that this cannot print out the info logs now.